lib/spack/spack/bootstrap/environment.py

```
@@ -92,7 +92,7 @@ def update_installations(self) -> None:
             tty.msg(f"[BOOTSTRAPPING] Installing dependencies ({', '.join(colorized_specs)})")
             self.write(regenerate=False)
             with tty.SuppressOutput(msg_enabled=log_enabled, warn_enabled=log_enabled):
-                self.install_all()
+                self.install_all(fail_fast=True)
```

Do we want/need this change for this PR? I'm not sure -- fail fast seems good to me for bootstrapping (especially parallel bootstrapping) but I am curious what @becker33 and @alalazo think.

---

I changed this while debugging and decided it was a better default -- maybe I should have factored it out as a separate PR but that's why I left it when I removed other debug statements.